src/code/z_effect_soft_sprite_dlftbls.c

```diff
@@ -0,0 +1,56 @@
+#include "global.h"
+#include "initvars.h"
+
+#define EFFECT_SS_OVERLAY(name)                                                                         \
+    {                                                                                                   \
+        (u32) _ovl_##name##SegmentRomStart, (u32)_ovl_##name##SegmentRomEnd, _ovl_##name##SegmentStart, \
+            _ovl_##name##SegmentEnd, NULL, &name##_InitVars, 1,                                         \
```

Comment:
I recommend using the macros for the Segments instead.
Also, is the `u32` cast required? Would this match with `uintptr_t` instead?
```suggestion
        SEGMENT_ROM_START(ovl_##name), SEGMENT_ROM_END(ovl_##name), SEGMENT_START(ovl_##name), \
            SEGMENT_END(ovl_##name), NULL, &name##_InitVars, 1,                                         \
```

Reply:
Ah those are better.
The casts are not needed.